drivers/can/can_stm32.c

```diff
@@ -52,6 +52,9 @@ static void can_stm32_get_msg_fifo(CAN_FIFOMailBox_TypeDef *mbox,
 	msg->dlc = mbox->RDTR & (CAN_RDT0R_DLC >> CAN_RDT0R_DLC_Pos);
```

Comment:
Typo in commit subject of "dricers: can: stm32: Implement timestamp for RX frames"
s/dricers/drivers/